go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/Azure/azure-event-hubs-go/v3 v3.5.0
 	github.com/containrrr/shoutrrr v0.7.1
 	github.com/fluxcd/notification-controller/api v1.0.0-rc.3
-	github.com/fluxcd/pkg/apis/event v0.4.1
+	github.com/fluxcd/pkg/apis/event v0.4.2-0.20230516071635-f4cf2dc1a8b0
```

Comment:
We are awaiting a controller-runtime release to pair with an update to Kubernetes 1.27.x, which would probably be best to land before tagging this.

Reply:
When is that expected to happen?

Reply:
Hopefully this week, as it is also blocking Kustomize v5 😬

Reply:
This has seen a release now.